examples/ping/ping.gpr

```diff
@@ -0,0 +1,48 @@
+project Ping is
+
+   for Languages use ("RecordFlux", "Python", "Ada", "C");
+   for Source_Dirs use (".", "specs", "generated", "src", "contrib");
+   for Object_Dir use "obj";
+   for Create_Missing_Dirs use "True";
+   for Main use ("ping.adb");
+
+   package Compiler is
+      for Driver ("recordflux") use "";
```

Comment:
Is the lower casing required? Otherwise I think we should use it consistently as it is used in `Languages`.

Reply:
It's not strictly necessary to have it lower-case. However, when GNAT Studio writes the project file, it will always convert it to lower-case (but not the `Languages` as you noticed). I'd leave it as is to reduce the noise when handling project files with GNAT Studio.